sql/2019/01_JavaScript/01_18.sql

```diff
@@ -0,0 +1,37 @@
+#standardSQL
+# 01_18-23: % of pages using JS APIs
+SELECT
+  client,
+  COUNT(0) AS total,
+  COUNTIF(atomics > 0) AS atomics,
+  ROUND(COUNTIF(atomics > 0) * 100 / COUNT(0), 2) AS pct_atomics,
+  COUNTIF(intl > 0) AS intl,
+  ROUND(COUNTIF(intl > 0) * 100 / COUNT(0), 2) AS pct_intl,
```

Comment:
should I be moving all of my query names to `pct_*`?

Reply:
Not necessary. As long as queries for a given chapter are internally consistent, that's mostly what matters. Especially since you're analyzing _and_ authoring, the naming shouldn't matter. It's just to give authors a unique ID to include/visualize the results.

In this case I suffix with the metric because there are multiple percent values in this query.